usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -164,44 +176,184 @@ def handle(self, *args, **options):
+        self.postgres_table_name = self.table_spec["swap_table"]
+        self.qualified_postgres_table = f"{postgres_schema}.{self.postgres_table_name}"
+        postgres_temp_schema = "temp"
+
+        # Determine the latest version of the Delta Table at the time this command is running. This will later
+        # be stored as the latest version written to the staging table, regardless of whether we update incrementally
+        # or repopulate the entire table
+        last_update_version = (
+            spark.sql(f"DESCRIBE HISTORY {self.qualified_source_delta_table}").select("version").first()[0]
+        )
+
+        updated_incrementally = False
+
+        if self.options["incremental"]:
+            # Validate that necessary TABLE_SPEC fields are present for incremental loads
+            if not ("incremental_delete_temp_schema" and "delta_table_load_version_key"):
+                self.logger.error(
+                    f"TABLE_SPEC configuration for {source_delta_table} is not sufficient for incremental loads"
+                )
+                self.logger.error(
+                    f"Values are required for fields: `incremental_delete_temp_schema` and `delta_table_load_version_key`"
+                )
+                raise ArgumentTypeError(
+                    f"TABLE_SPEC configuration for {source_delta_table} is not sufficient for incremental loads"
+                )
+
+            # Retrieve information about which versions of the Delta Table have been loaded where. We'll use
+            # the `last_live_version` to query changes because we want all the data that has not persisted
+            # to the live table, regardless of what has been previously written to the staging table because it
+            # will be overwritten.
+            last_staging_version, last_live_version = get_last_delta_table_load_versions(delta_table_load_version_key)
+
+            # Create a Dataframe with a unique list of entities (based on the table's unique key) that have been
+            # updated since this command was last run. If an entity has been updated multiple times, the latest
+            # update will be selected. This utilizes Delta Lake's Change Data Feed feature.
+            # This will be used to determine if the number of updated records exceeds the update threshold, and
+            # (if not exceeding) it will serve as the basis for dataframes to be written to Postgres with incremental
+            # changes.
+            unique_identifiers = self.table_spec["unique_identifiers"]
+            distinct_df = spark.sql(
+                f"""
+                SELECT * FROM (
+                    SELECT * ,
+                    ROW_NUMBER() OVER (PARTITION BY {', '.join(unique_identifiers)} ORDER BY _commit_version DESC) AS row_num
+                    FROM table_changes('{self.qualified_source_delta_table}', {last_live_version + 1})
+                    WHERE _change_type in ('insert', 'update_postimage', 'delete')
```

Comment:
This SQL does not follow the agreed upon team standards for style. There are a few key words in here that are not in all caps.

Reply:
Updated `in` to `IN` I didn't see any more. Let me know if I missed anything.